server/routes/creatingReports/incidentDetails.ts

```diff
@@ -132,6 +133,17 @@ export default class IncidentDetailsRoutes {
       incidentDate ? incidentDate.value : null
     )
 
+    if (incidentDate) {
+      const duplicates = await this.draftReportService.getPotentialDuplicates(
+        parseInt(bookingId, 10),
+        moment(incidentDate.value),
+        token
+      )
+
+      if (duplicates?.length) {
```

Comment:
this will never be undefined so shouldn't need the `?`

Reply:
done